### ISSUE TITLE
Infinite scroll blog page

### ISSUE THREAD

**Issue by kylepolich** [data-skeptic/dataskeptic.com#695]

**Comment by gliblozhkovyi** [comment]
Just a simple note: current infinite scroll component is scalable, and this shouldn't take time in case API will have the same format. 

In other words, blogs and podcast page will work in the same way for an end user

**Comment by kylepolich** [comment]
@gliblozhkovyi yes, good point.  I was thinking of it very different because of historical reasons, but actually, /podcast and /blog is the same idea.

I think we have offset and limit for the blogs API already.  Do you have what you need or do I need to give you something better?

**Comment by gliblozhkovyi** [comment]
I checked the api response format again (/prod/blog/list?limit=10&offset=0&prefix=)

Could it be the same as for `/podcasts`? 

I mean the structure like this one:
```
{
   "blogs": [...],
   "total": N
}

```
